x/auth/ante/ante.go

```diff
@@ -39,7 +39,6 @@ func NewAnteHandler(options HandlerOptions) (sdk.AnteHandler, error) {
 	}
 
 	anteDecorators := []sdk.AnteDecorator{
-		NewSetUpContextDecorator(), // outermost AnteDecorator. SetUpContext must be called first
```

Comment:
This is the only antehandler migrated in this PR, it's replaced by the `GasMiddleware`.

I migrated this antehandler (and not any other) because it was needed for the `RecoveryMiddleware`